gradoop-examples/src/main/java/org/gradoop/utils/sampling/RandomEdgeSamplingRunner.java

```diff
@@ -41,7 +40,7 @@
   public static void main(String[] args) throws Exception {
     LogicalGraph graph = readLogicalGraph(args[0], args[1]);
     LogicalGraph sample = graph.callForGraph(new RandomEdgeSampling(Float.parseFloat(args[2])));
-    new CSVDataSink(args[3], args[0] + "/metadata.csv", graph.getConfig()).write(sample);
+    writeLogicalGraph(sample,args[0]);
```

Comment:
writeLogicalGraph would write your output by default as json. I would suggest to add another parameter witch defines the output type (see GradoopFormatConverter for example).